py3-pip.yaml

```diff
@@ -32,6 +32,17 @@ pipeline:
       expected-commit: 97146c7f4cd85551f3dc261830a57f304e43c181
       tag: ${{package.version}}
 
+  - uses: patch
+    with:
+      patches: bump-urllib3.patch
+
+  - runs: |
+      rm -rf ./src/pip/_vendor/urllib3
+      git clone https://github.com/urllib3/urllib3.git /home/urllib3
+      cd /home/urllib3
+      git checkout d9d85c88aa644af56d5e129634e750ce76e1a765
+      mv  ./src/urllib3 ../build/src/pip/_vendor/
+
```

Comment:
you'll need to rerun the vendoring. I suggest you do that and use the patch from that here as we don't have nox installed